cypress/e2e/spec.cy.js

```diff
@@ -11,11 +13,20 @@ describe("Approve all files and download ZIP", () => {
       cy.get("[data-sacro-el='outputDetailsBtnApprove']").click();
     });
 
+    // open the modal, comment, and submit the form
+    cy.get("button#openModalBtn").click();
+    cy.get("[data-sacro-el='submissionComment']").type(
+      "Everything has been approved"
+    );
+    cy.get("#approveForm").submit();
+
+    cy.url().should("eq", "http://localhost:8000/review/current/");
+    //
     // prepare for form submission that returns back a file
     // https://on.cypress.io/intercept
     cy.intercept(
       {
-        pathname: "/review*",
+        pathname: "/review/current/approved-outputs/",
```

Comment:
Might be good to do something similar here for downloading the review summary and checking that a text file is returned, assuming that's quite straightforward to do.

Reply:
Done.  I wasn't sure how to simplify the `oneOf` for a single item so I've left it as is.